### PR TITLE
Game scoring

### PR DESCRIPTION
**Pull request by iangornall**
Score is working on frontend.  Timer also shows.  Score is reduced by 10 points when an incorrect answer is chosen.  Score and timer reset on each level.

### CODE REVIEW

File: public/js/game.js [hunk 1]
```diff
@@ -114,6 +128,8 @@ interact('.play-color').dropzone({
       event.relatedTarget.classList.add('hidden');
       event.target.setAttribute('data-target', false)
       event.relatedTarget.setAttribute('data-target', false)
+      clearInterval(scoreTimer);
+      clearInterval(timeTimer);
       setTimeout(() => {
         level++;
         if (level > 4) {
```
Comment (sarathrasher) [inline public/js/game.js:135]: So it tries to get level data if the level is above 4, but we don't actually have any more than 4 levels. Should this be <= 4? Or am I missing something?

File: public/js/game.js [hunk 2]
```diff
@@ -38,10 +41,19 @@ let getLevelData = level => {
...
+        scoreDisplay.textContent = score;
+      }
+    }, 200);
+    // Starting time
+    let startTime = Date.now();
+    timeTimer = setInterval(() => {
+      time = Date.now() - startTime;
+      let date = new Date(time);
+      timeDisplay.textContent = date.getSeconds();
+    }, 200);
```
Comment (sarathrasher) [inline public/js/game.js:56]: So this is counting down every 200 ms, but didn't we talk about counting down every second?

**Review by sarathrasher — COMMENTED** [review]
See comments. One is from an older commit but I just wanted clarification on it

File: public/js/game.js (hunk 1 above)
Comment (iangornall) [reply public/js/game.js:135]: the next line says to console.log game over.  I haven't handled that yet.

File: public/js/game.js (hunk 2 above)
Comment (iangornall) [reply public/js/game.js:56]: I don't recall setting a time to decrement the score.  This works pretty well for a game timer.  If the score goes down every 200ms, then its down 5 points in 1 second, still leaving 20 seconds in a level before it hits zero.

Comment (sarathrasher) [reply public/js/game.js:56]: Alrighty that sounds good. 20 seconds does sound like a reasonable amount of time.

File: public/js/game.js [hunk 3]
```diff
@@ -123,9 +139,18 @@ interact('.play-color').dropzone({
         getLevelData(level);
```
Comment (sarathrasher) [inline public/js/game.js]: Correct me if I'm wrong, but on what I'm looking at it says the next line is getLevelData(level).

Comment (iangornall) [reply public/js/game.js]: that's line 139, so it's not showing you every line of code.  Click the blue bar in between.

File: public/js/game.js [hunk 4]
```diff
@@ -1,7 +1,10 @@
 let playColors = document.querySelectorAll('.play-color');
 let level = 1;
 let score = 100;
+let time;
 let scoreDisplay = document.querySelector('.score');
+let timeDisplay = document.querySelector('.time');
+let scoreTimer;
 let getLevelData = level => {
```
Comment (sarathrasher) [inline public/js/game.js]: If there's another name for this function that would help eliminate confusion. I know it does get level data. But maybe there's a more specific name we could change it to?

**Review by sarathrasher — APPROVED** [review]

**Pull request merged**